fast-open/client/client-handle-initial-cookie-of-length-16.pkt

```diff
@@ -60,7 +60,7 @@
 +0.00 bind(3, ..., ...) = 0
 +0.00 setsockopt(3, IPPROTO_TCP, TCP_FASTOPEN, [1], 4) = 0
 +0.00 sendto(3, ..., 100, 0, ..., ...) = 100
-+0.00 > S  0:100(100) win 65535 <mss 1460,nop,wscale 6,sackOK,TS val 0 ecr 0,nop,nop,FO 01020304050607080910111213141516>
++0.00 > S  0:100(100) win 65535 <mss 1460,nop,wscale 6,sackOK,TS val 0 ecr 0,FO 01020304050607080910111213141516,eol,eol>
```

Comment:
Isn't this already in the repo as https://github.com/freebsd-net/tcp-testsuite/commit/45808d22e4a9333d989fbd6d947fd75d06904999?